src/guide/components/props.md

```diff
@@ -423,7 +423,7 @@ defineProps({
 ```
 
 :::tip
-`defineProps()` の引数の中のコードは、コンパイル時に式全体が外側の関数スコープに移されるため、`**<script setup>`** 内で宣言している他の変数にアクセスすることができません。
+`defineProps()` の引数の中のコードは、コンパイル時に式全体が外側の関数スコープに移されるため、**`<script setup>` 内で宣言している他の変数にアクセスすることができません** 。
```

Comment:
原文：

https://github.com/vuejs/docs/blob/0fc28808ee113261ea0b5bc050b1a36eb5b97549/src/guide/components/props.md?plain=1#L426


> ```Code inside the `defineProps()` argument **cannot access other variables declared in `<script setup>`**, because the entire expression is moved to an outer function scope when compiled.```